docker-compose.yml

```diff
@@ -27,6 +27,17 @@ services:
       - elasticsearch
     command: /bin/sh -c './bin/rails s -b 0.0.0.0'
 
+  selenium:
+    build:
+      context: .
+      dockerfile: Dockerfile.selenium
```

Comment:
I prefer the traditional `name` + `ext` scheme for file names: `selenium.dockerfile`

Also wondering whether we could use a pre-built image for selenium. Did you look into that?

Reply:
Yeah, they weight 1GB with lots of useless junk, versus 380MB for a custom Alpine image :shrug: